GIFrameworkMaps.Web/Views/ManagementAnalytics/Create.cshtml

```diff
@@ -0,0 +1,68 @@
+}
+
+<h1>@ViewData["Title"]</h1>
+@using (Html.BeginForm("Create", "ManagementAnalytics",null, FormMethod.Post, true, null))
+{
+    <div class="row mt-4">
+        <div class="col-md-6">
+            <div asp-validation-summary="ModelOnly" class="alert alert-danger mb-3"></div>
+            <div class="mb-3">
+                <label asp-for="analyticDefinition.ProductName" class="control-label"></label>
+                <select asp-for="analyticDefinition.ProductName" asp-items="Model.availableProducts" class="form-select">
+                    <option value="">Please select</option>
+                </select>
+                <span asp-validation-for="analyticDefinition.ProductName" class="text-danger"></span>
+            </div>
+            <div class="mb-3">
+                <label asp-for="analyticDefinition.ProductKey" class="control-label"></label>
+                <input asp-for="analyticDefinition.ProductKey" class="form-control" />
+                <span asp-validation-for="analyticDefinition.ProductKey" class="text-danger"></span>
+            </div>
+            <div class="mb-3">
+                <label asp-for="analyticDefinition.CookieControl" class="control-label"></label>
+                <select asp-for="analyticDefinition.CookieControl" asp-items="Model.availableCookieControl" class="form-select">
+                    <option value="">None</option>
+                </select>
+                <span asp-validation-for="analyticDefinition.CookieControl" class="text-danger"></span>
+            </div>
+            <div class="mb-3">
+                <label asp-for="analyticDefinition.Enabled" class="control-label"></label>
+                <input asp-for="analyticDefinition.Enabled" class="form-check-input" type="checkbox" />
+                <span asp-validation-for="analyticDefinition.Enabled" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="card mb-2">
+                <div class="card-header">
+                    Versions
+                </div>
+                <div class="card-body">
+                    @if (Model.SelectedVersions != null && Model.SelectedVersions.Count == 0)
+                    {
+                        <p class="card-text">Will apply to all versions unless you select from the list below.</p>
+                    }
+                    <partial name="ManagementPartials/VersionAnalyticList" model="Model.AvailableVersions.ToList()" view-data="ViewData" />
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row mt-4">
+        <div class="col-md-6">
+            <div class="mb-3">
+                <input type="hidden" asp-for="analyticDefinition.Id" />
+                <input type="hidden" asp-for="analyticDefinition.DateModified" />
+                <div class="mb-3">
+                    <input type="submit" value="Add Record" class="btn btn-primary" />
```

Comment:
To stay consistent with the other management pages, can we change the button test to say 'Create' rather then 'Add record'